src/pkg/mcp/setup.go

```diff
@@ -296,28 +294,37 @@ func handleVSCodeConfig(configPath string) error {
 
 func handleStandardConfig(configPath string) error {
 	// For all other clients, use the standard format
+	var existingData map[string]any
 	var config MCPConfig
 
 	// Check if the file exists
 	if _, err := os.Stat(configPath); err == nil {
 		// File exists, read it
 		data, err := os.ReadFile(configPath)
```

Comment:
Remove the `os.Stat` and just read, see https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use